internal/auth/oidc/repository_auth_method.go

```diff
@@ -196,7 +196,7 @@ func upsertOplog(ctx context.Context, w db.Writer, oplogWrapper wrapping.Wrapper
 	if err != nil {
 		return errors.Wrap(err, op, errors.WithMsg("unable to get ticket"))
 	}
-	metadata := acct.oplog(operation, scopeId)
+	metadata := acct.oplog(operation)
```

Comment:
please add scope id back into the metadata. (see my earlier comment)

Reply:
Added. Thanks!